examples/pxScene2d/src/node_modules/console_wrap.js

```diff
@@ -0,0 +1,60 @@
+/*
+
+pxCore Copyright 2005-2018 John Robinson
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+"use strict";
+
+/**
+ * convert multiple messages to single string
+ * @param  {...any} args the messages
+ */
+var console = require("console");
+function stringify(...args) {
+    return Array.prototype.slice.apply(arguments).join(' ');
+}
+
+function trace(...args) {
+    console.trace("[SPARK TRACE] " + stringify(...args));
+}
+
+function warn(...args) {
+    console.warn("[SPARK WARN] " + stringify(...args));
+}
+
+function error(...args) {
+    console.error("[SPARK ERROR] " + stringify(...args));
+}
+
+function log(...args) {
+    console.log("[SPARK LOG] " + stringify(...args));
+}
+
+function info(...args) {
+    console.info("[SPARK INFO] " + stringify(...args));
+}
+
+module.exports = {
+    trace: trace,
+    warn: warn,
+    error: error,
+    log: log,
+    info: info,
+    dir: console.dir,
+    assert: console.assert,
+    time: console.time,
+    timeEnd: console.timeEnd
```

Comment:
How about other console api functionality?  It looks like it will now be lost.  It's true that we may not be using it today, but if an app wishes to access other apis in console in the future then it will require a change to the console wrapper.

Reply:
verified and made it in sync with console api from nodejs